### PR TITLE
Create default program when new Org

### PR DESCRIPTION
**Pull request by Menda**
## Purpose

In certain user cases it may be necessary to have at least one Program to create other objects and keep certain relationships. By default this feature is deactivated.

Related ticket: https://github.com/Humanitec/ActivityAPI/issues/108

### CODE REVIEW

File: workflow/signals.py [hunk 1]
```diff
@@ -62,6 +62,15 @@ def add_users_to_default_wflvl1(sender, instance, **kwargs):
                                     workflowlevel1=wflvl1_1)
 
 
+@receiver(signals.post_save, sender=Organization)
+def create_default_program(sender, instance, **kwargs):
```
Comment (jefmoura) [inline workflow/signals.py]: You'll create a default program when the organization is updated as well. Check if the `kwargs.get('created')` is `False`.

File: workflow/tests/test_signals.py [hunk 2]
```diff
@@ -102,6 +103,21 @@ def test_demo_workflowteam_assignment_not_reassigned_on_update(
         self.assertEqual(num_results, 1)
 
 
+@tag('pkg')
+class CreateDefaultProgramTest(TestCase):
```
Comment (jefmoura) [inline workflow/tests/test_signals.py]: Create a new test case to make sure that you're not creating programs when you update the org.

Comment (Menda) [reply workflow/tests/test_signals.py]: Stupid me, true. I just fixed it

**Comment by jefmoura** [comment]
@Menda don't forget to add the variable `CREATE_DEFAULT_PROGRAM` in the deployment repo.

**Comment by Menda** [comment]
Yes

**Pull request merged**